packages/server/src/index.test.ts

```diff
@@ -28,7 +28,7 @@ jest.mock('pg', () => {
       if (sql === 'SELECT "version" FROM "DatabaseMigration"') {
         return { rows: [{ version: 1000000 }] };
       }
-      if (sql === 'SELECT "User"."id", "User"."content" FROM "User" WHERE "deleted"=$1 LIMIT 1') {
+      if (sql === 'SELECT "User"."id", "User"."content" FROM "User" WHERE "User"."deleted"=$1 LIMIT 2') {
```

Comment:
This is the query to check "is the database seeded".  The existing query was out of date, so the test would unnecessarily re-seed the database.  By fixing this line, the test went from 52 seconds to 3 seconds.